ondemand.osc.edu/apps/bc_desktop/submit/slurm.yml.erb

```diff
@@ -0,0 +1,5 @@
+---
+script:
+  native:
+    - "-N"
+    - "<%= bc_num_slots.blank? ? 1 : bc_num_slots.to_i %>"
```

Comment:
`node_type` from the `form.yml` isn't used here (and is in torque parlance).  There are no `vis` nodes in SLURM yet so we can get rid of those and you can request up to 4 GPUs.

You can the submit on my branch to jupyter for what arguements to use. 

https://github.com/OSC/bc_osc_jupyter/blob/add-pitzer-exp/submit.yml.erb

I'm just saying this, but I feel like we should use the long form `--nodes` and `--ntasks-per-node` to start just because they're more explicit than `-N` or whatever the short argument for GPU is.